src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -360,6 +363,47 @@ def _geotiff_filename(self, inter_filename):
 
         return f"{tile_filename}_{band_idx}_{band_name}.tif"
 
+    def _browse_filename(self, inter_filename):
+        """
+        Returns the file name to use for the PNG browse image produced by
+        the DSWx-S1 PGE.
+
+        The browse image filename for the DSWx-S1 PGE consists of:
+
+            <Tile filename>_BROWSE.png
+
+        Where <Tile filename> is returned by DSWxS1PostProcessorMixin._tile_filename().
+
+        Parameters
+        ----------
+        inter_filename : str
+            The intermediate filename of the output browse image to generate a
+            filename for. This parameter may be used to inspect the file in order
```

Comment:
This is a personal on about ending a sentence with a preposition.  Personally I would rewrite it: "The intermediate filename of the output browse image from which to generate a filename."